sql-cli/sql_cli/run_dag.py

```diff
@@ -149,6 +149,10 @@ def _run_task(ti: TaskInstance, session: Session) -> None:
         session.flush()
         session.commit()
         pprint("[bold green]SUCCESS[/bold green]")
+    except OperationalError as operational_exception:
+        pprint("[bold red]FAILED[/bold red]")
+        original_message = operational_exception.orig.args[0]
+        pprint(f"  Error [red]{original_message}[/red] using connection [bold]{ti.task.conn_id}[/bold].")
```

Comment:
@tatiana I think it is better to print the original message and adding a bit more information to it - rather than us reading the message (string) and parsing the information out of it. If we want to do that that could cause a massive overhead we would have to maintain.

Reply:
<img width="543" alt="Screenshot 2022-10-20 at 11 17 20" src="https://user-images.githubusercontent.com/23282078/196908714-ec2f715f-564d-4fc1-895d-4a6091232451.png">

WDYT?

Reply:
Sounds great, @feluelle , as long as we avoid massive Python tracebacks and keep these messages small and straightforward, it is great!